<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>792</width>
    <height>446</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>792</width>
    <height>446</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>792</width>
    <height>446</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>医院病房护理智能无线呼叫及管理系统</string>
  </property>
  <property name="windowIcon">
   <iconset resource="src/src.qrc">
    <normaloff>:/new/prefix1/64X64.ico</normaloff>:/new/prefix1/64X64.ico</iconset>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>792</width>
     <height>446</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string/>
   </property>
   <property name="tabShape">
    <enum>QTabWidget::Rounded</enum>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <property name="elideMode">
    <enum>Qt::ElideNone</enum>
   </property>
   <widget class="QWidget" name="SettingTab">
    <property name="toolTip">
     <string/>
    </property>
    <attribute name="title">
     <string>  设 置  </string>
    </attribute>
    <widget class="QLineEdit" name="IP_lineEdit">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>231</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Sawasdee</family>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;服务器 IP：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QToolButton" name="Connect_toolButton">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>120</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>连接</string>
     </property>
    </widget>
    <widget class="QToolButton" name="Disconnect_toolButton">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>120</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>断开</string>
     </property>
    </widget>
    <widget class="QToolButton" name="toolButton_1">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>200</y>
       <width>51</width>
       <height>41</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>51</width>
       <height>41</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>51</width>
       <height>41</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Sawasdee</family>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>1</string>
     </property>
    </widget>
    <widget class="QToolButton" name="toolButton_2">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>200</y>
       <width>51</width>
       <height>41</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>51</width>
       <height>41</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>51</width>
       <height>41</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Sawasdee</family>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>2</string>
     </property>
    </widget>
    <widget class="QToolButton" name="toolButton_3">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>200</y>
       <width>51</width>
       <height>41</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>51</width>
       <height>41</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>51</width>
       <height>41</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Sawasdee</family>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>3</string>
     </property>
    </widget>
    <widget class="QToolButton" name="toolButton_4">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>200</y>
       <width>51</width>
       <height>41</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>51</width>
       <height>41</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>51</width>
       <height>41</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Sawasdee</family>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>4</string>
     </property>
    </widget>
    <widget class="QToolButton" name="toolButton_5">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>250</y>
       <width>51</width>
       <height>41</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>51</width>
       <height>41</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>51</width>
       <height>41</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Sawasdee</family>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>5</string>
     </property>
    </widget>
    <widget class="QToolButton" name="toolButton_6">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>250</y>
       <width>51</width>
       <height>41</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>51</width>
       <height>41</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>51</width>
       <height>41</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Sawasdee</family>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>6</string>
     </property>
    </widget>
    <widget class="QToolButton" name="toolButton_7">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>250</y>
       <width>51</width>
       <height>41</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>51</width>
       <height>41</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>51</width>
       <height>41</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Sawasdee</family>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>7</string>
     </property>
    </widget>
    <widget class="QToolButton" name="toolButton_8">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>250</y>
       <width>51</width>
       <height>41</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>51</width>
       <height>41</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>51</width>
       <height>41</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Sawasdee</family>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>8</string>
     </property>
    </widget>
    <widget class="QToolButton" name="toolButton_9">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>300</y>
       <width>51</width>
       <height>41</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>51</width>
       <height>41</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>51</width>
       <height>41</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Sawasdee</family>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>9</string>
     </property>
    </widget>
    <widget class="QToolButton" name="toolButton_10">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>300</y>
       <width>51</width>
       <height>41</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>51</width>
       <height>41</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>51</width>
       <height>41</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Sawasdee</family>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QToolButton" name="toolButton_11">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>300</y>
       <width>51</width>
       <height>41</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>51</width>
       <height>41</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>51</width>
       <height>41</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Sawasdee</family>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>.</string>
     </property>
    </widget>
    <widget class="QToolButton" name="toolButton_12">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>350</y>
       <width>231</width>
       <height>41</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>51</width>
       <height>41</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>500</width>
       <height>200</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Sawasdee</family>
       <pointsize>13</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>DEL</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>20</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;串口设置：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QComboBox" name="COM_comboBox">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>60</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <item>
      <property name="text">
       <string>ttyUSB0</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>ttyUSB1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>ttyUSB2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>ttyUSB3</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>ttyUSB4</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>ttyUSB5</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="Bote_comboBox">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>120</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <item>
      <property name="text">
       <string>9600</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>38400</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>115200</string>
      </property>
     </item>
    </widget>
    <widget class="QToolButton" name="Open_toolButton">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>200</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>打开</string>
     </property>
    </widget>
    <widget class="QToolButton" name="Close_toolButton">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>260</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>关闭</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEdit">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>30</y>
       <width>111</width>
       <height>371</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QToolButton" name="Clear_toolButton">
     <property name="geometry">
      <rect>
       <x>680</x>
       <y>140</y>
       <width>81</width>
       <height>121</height>
      </rect>
     </property>
     <property name="text">
      <string>Clear</string>
     </property>
    </widget>
    <widget class="QToolButton" name="toolButton_13">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>300</y>
       <width>51</width>
       <height>41</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>51</width>
       <height>41</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>51</width>
       <height>41</height>
      </size>
     </property>
     <property name="text">
      <string>:</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="PatientTab">
    <attribute name="title">
     <string>  节 点  </string>
    </attribute>
    <widget class="QToolButton" name="Patient_toolButton_1">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>20</y>
       <width>91</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>1号节点</string>
     </property>
    </widget>
    <widget class="QToolButton" name="Patient_toolButton_2">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>20</y>
       <width>91</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>2号节点</string>
     </property>
    </widget>
    <widget class="QToolButton" name="Patient_toolButton_3">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>20</y>
       <width>91</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>3号节点</string>
     </property>
    </widget>
    <widget class="QToolButton" name="Patient_toolButton_4">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>20</y>
       <width>91</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>4号节点</string>
     </property>
    </widget>
    <widget class="QToolButton" name="Patient_toolButton_6">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>20</y>
       <width>91</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>6号节点</string>
     </property>
    </widget>
    <widget class="QToolButton" name="Patient_toolButton_5">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>20</y>
       <width>91</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>5号节点</string>
     </property>
    </widget>
    <widget class="QToolButton" name="Patient_toolButton_7">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>110</y>
       <width>91</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>7号节点</string>
     </property>
    </widget>
    <widget class="QToolButton" name="Patient_toolButton_8">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>110</y>
       <width>91</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>8号节点</string>
     </property>
    </widget>
    <widget class="QToolButton" name="Patient_toolButton_9">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>110</y>
       <width>91</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>9号节点</string>
     </property>
    </widget>
    <widget class="QToolButton" name="Patient_toolButton_10">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>110</y>
       <width>91</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>10号节点</string>
     </property>
    </widget>
    <widget class="QToolButton" name="Patient_toolButton_11">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>110</y>
       <width>91</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>11号节点</string>
     </property>
    </widget>
    <widget class="QToolButton" name="Patient_toolButton_12">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>110</y>
       <width>91</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>12号节点</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="src/src.qrc"/>
 </resources>
 <connections/>
</ui>
